src/tech_it_easy.sql

```diff
@@ -0,0 +1,70 @@
+drop table if exists wall_brackets;
+drop table if exists ci_modules;
+drop table if exists remote_controllers;
+drop table if exists televisions;
```

Comment:
Je hebt dit in de goede volgorde gezet, zodat je geen CASCADE hoeft te gebruiken